docker/setup_demo.sh

```diff
@@ -19,6 +19,9 @@
 SCRIPT_PATH=$(cd `dirname $0`; pwd)
 HUDI_DEMO_ENV=$1
 WS_ROOT=`dirname $SCRIPT_PATH`
+cd hoodie/hadoop
+mvn clean package -DskipTests
```

Comment:
Why do we need this here?  Should mvn build of hudi repo using `-Pintegration-tests` before running this script solve the problem?  It’s not a good idea to couple the docker setup with the build.

Reply:
+1